File: 2427-number-of-common-factors/2427-number-of-common-factors.kt
class Solution {
    fun commonFactors(a: Int, b: Int): Int {
        var res = 0
        for (i in 1..(Math.min(a, b))) {
            if (a % i == 0 && b % i == 0) {
                res += 1
            }
        }
        return res
    }
}

File: 2427-number-of-common-factors/2427-number-of-common-factors.js
/**
 * @param {number} a
 * @param {number} b
 * @return {number}
 */
var commonFactors = function(a, b) {
    let res = 0
    for (let i=1; i <= Math.min(a, b); i++) {
        if (!(a % i) && !(b % i)) res++
    }
    return res
};